classes/controller/manage.php

```diff
@@ -78,20 +78,34 @@ public function action_edit($id = false)
 	 */
 	public function action_delete($id = false)
 	{
-		$message = \Ticker\Model_Message::find($id);
-
-		// try to delete the message
-		try
-		{
-			$message->delete();
-			\Session::set_flash("success", "Message deleted");
-
-			\Response::redirect(\Uri::create("/admin/ticker/manage"));
+		$redirect_url = \Uri::create('/admin/ticker/manage');
+
+		// Check if ID is valid
+		if ($id = (int)$id) {
```

Comment:
This ```if()``` is not necessary. ORM will sanitize and find() will validate input.

If we want validation here, then only positive integers should pass. This will accept values like '1aa' or '11.12'.